src/components/DocumentSelector/index.js

```diff
@@ -1,13 +1,14 @@
 // @flow
-import * as React from 'react'
```

Comment:
Interesting. I was under the impression that the React import was required for Flow to work. Seem to recall seeing some errors when it wasn't. But it looks like it actually is not required.